src/fides/api/ctl/routes/crud.py

```diff
@@ -60,8 +62,9 @@
         },
     )
     async def create(
-        resource: resource_model,
+        resource: fides_model,
         resource_type: str = get_resource_type(router),
+        db: AsyncSession = Depends(get_async_db),
```

Comment:
`get_async_db` is a dependency here that creates a new AsyncSession that is used in this request, and closed when finished. https://fastapi.tiangolo.com/tutorial/sql-databases/#create-a-dependency

This pattern is used frequently on the ops side.

Reply:
I believe this is a much better pattern, thank you for bringing this over!